mayastor/src/bdev/dev/nvmx/device.rs

```diff
@@ -119,6 +120,17 @@ impl NvmeBlockDevice {
             name: String::from(name),
         }
     }
+
+    fn get_controller(
+        &self,
+    ) -> Result<Arc<Mutex<NvmeController<'static>>>, CoreError> {
```

Comment:
'static? 🤔

Reply:
Do we want the returned NVMe controller to last possibly the entire life of the binary run?

Reply:
Indeed we can move controller lookup close to the caller and get rid of get_controller(), as its used only once.